inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -64,6 +64,32 @@ void LayerTestsCommon::Compare(const std::vector<std::uint8_t> &expected, const
     }
 }
 
+void LayerTestsCommon::Compare(const InferenceEngine::Blob::Ptr &expected, const InferenceEngine::Blob::Ptr &actual) {
```

Comment:
looks like the function should be in `data_utils`

Reply:
you are right. `Compare` function should be in utils scope. But currently it uses `threshold` field of test class.
And this refactoring is out of scope of this PR.